server/src/test/scala/za/co/absa/atum/server/api/service/PartitioningServiceUnitTests.scala

```diff
@@ -139,6 +143,18 @@ object PartitioningServiceUnitTests extends ZIOSpecDefault with TestData {
             failsWithA[ServiceError]
           )
         }
+      ),
+      suite("GetPartitioningByIDSuite")(
+        test("Returns expected Right with PartitioningWithIdDTO") {
+          for {
+            result <- PartitioningService.getPartitioning(1111L)
+          } yield assertTrue(result == partitioningWithIdDTO1)
+        },
+        test("Returns expected ServiceError") {
```

Comment:
The first test case is OK as you are simulating successful scenario. The other two test cases should be for unexpected error with `ZIO.fail(GeneralDatabaseError("boom!"))` and assert on `GeneralServiceError` and third test case should be for `ZIO.fail(NotFoundDatabaseError("Partitioning not found"))` and assert on `NotFoundServiceError`.